config/routes.rb

```diff
@@ -30,6 +30,20 @@
       post "content_improvement", to: "content_improvement#create", defaults: { format: :js }
       resources "page_improvements", only: [:create], format: false
     end
+
+    get "govuk-app", to: "govuk_app#new"
```

Comment:
You could move these inside the namespace, I think?

```
    namespace :govuk_app, path: "govuk-app" do
      get "/", to: "govuk_app#new"
      post "/", to: "govuk_app#create"
      get "/confirmation", to: "govuk_app#confirmation"

      get "problem-reports", to: "problem_reports#new"
      post "problem-reports", to: "problem_reports#create"

      get "suggestions", to: "suggestions#new"
      post "suggestions", to: "suggestions#create"
    end
```

Reply:
I tried this initially but was met with a load of routing errors, so separated the routes out, similarly to the contact web routes.